gobblin-modules/gobblin-kafka-common/src/main/java/org/apache/gobblin/source/extractor/extract/kafka/KafkaExtractorStatsTracker.java

```diff
@@ -249,6 +263,7 @@ public void updateStatisticsForCurrentPartition(int partitionIdx, long readStart
     this.workUnitState.setProp(KafkaUtils.getPartitionPropName(KafkaSource.STOP_FETCH_EPOCH_TIME, partitionId),
         Long.toString(stats.getStopFetchEpochTime()));
     tagsForPartition.put(PROCESSED_RECORD_COUNT, Long.toString(stats.getProcessedRecordCount()));
+    tagsForPartition.put(SLA_MISSED_RECORD_COUNT, Long.toString(stats.getSlaMissedRecordCount()));
```

Comment:
If `SLA_MISSED_RECORD_COUNT ` is added anyway, how to distinguish between non-SLA-miss versus non-configured for whoever just monitoring the dashboard?

Reply:
Good point. I initialized the SLA_MISSED_RECORD_COUNT to -1. In case, SLA is configured the count should be >= 0.